src/components/BCHeader/styles.scss

```diff
@@ -15,6 +19,35 @@ a.header-link:link {
   font-weight: bold;
 }
 
-.#{vars.$bcgov-prefix}--header__global .#{vars.$bcgov-prefix}--btn--icon-only.#{vars.$bcgov-prefix}--header__action svg {
+.#{vars.$bcgov-prefix}--side-nav__items {
+  display: flex;
+  flex-direction: column;
+}
+
+.support-section {
+  margin-top: auto;
+}
+
+.#{vars.$bcgov-prefix}--header__global .#{vars.$bcgov-prefix}--btn--icon-only.#{vars.$bcgov-prefix}--header__action svg,
+button.#{vars.$bcgov-prefix}--header__action svg {
   fill: colors.$white;
-}
\ No newline at end of file
+}
+
+button.#{vars.$bcgov-prefix}--header__action.#{vars.$bcgov-prefix}--header__menu-trigger {
+  border-radius: 4px;
+}
+
+button.#{vars.$bcgov-prefix}--header__action.#{vars.$bcgov-prefix}--header__menu-trigger:focus {
+  border-color: colors.$white;
+}
+
+button.#{vars.$bcgov-prefix}--header__action.#{vars.$bcgov-prefix}--header__action--active svg {
+  fill: colors.$gray-100;
+}
+
+/* Small - Up to 320px */
+@media only screen and (max-width: 672px) {
```

Comment:
Up to 320px? Or 672px?

Reply:
672, the commentary is wrong, already fixed it.